force-app/main/default/lwc/errorPanel/templates/noDataIllustration.html

```diff
@@ -126,15 +126,15 @@
                     <path
                         vector-effect="non-scaling-stroke"
                         class="slds-illustration__fill-secondary"
-                        d="M18.9209988,1.95433401 L33.259296,51.443436 C33.5666778,52.5043744 32.9557995,53.613617 31.8948612,53.9209988 C31.7139843,53.9734036 31.5266126,54 31.3382972,54 L2.6617028,54 C1.5571333,54 0.661702805,53.1045695 0.661702805,52 C0.661702805,51.8116846 0.688299176,51.6243129 0.74070397,51.443436 L15.0790012,1.95433401 C15.386383,0.893395645 16.4956256,0.282517358 17.556564,0.589899164 C18.2152102,0.780726338 18.7301717,1.29568777 18.9209988,1.95433401 Z"
+                        d="M18.9209988,1.95433401 L33.259296,51.443436 C33.5666778,52.5043744 32.9557995,53.613617 31.8948612,53.9209988 C31.7139843,53.9734036 31.5266126,54 31.3382972,54 L2.6617028,54 C1.5571333,55.0.661702805,53.1045695 0.661702805,52 C0.661702805,51.8116846 0.688299176,51.6243129 0.74070397,51.443436 L15.0790012,1.95433401 C15.386383,0.893395645 16.4956256,0.282517358 17.556564,0.589899164 C18.2152102,0.780726338 18.7301717,1.29568777 18.9209988,1.95433401 Z"
```

Comment:
Thank you for the review @pozil! I'll be sure to address these in the PR.

I just wanted to check in about the numerous random text, ID number, etc. changes, as I only committed the 4~5 files I worked on. I mentioned the issue to @schandlergarcia, who'd helped me debug the test runner, when we chatted just now, and he was curious which branch I'm merging into; should it be something other than `main`? Is that part of the issue?

Reply:
I think that the problem is that you created your branch from @albarivas' branch. Alba's branch got merged into `prerelease/summer22` branch then `prerelease/summer22` got merged into `main` when the release went out (Monday 13). This means that your PR includes some the changes that were pushed by Alba. You probably want to rebase it.

As a general rule, we work of `main` however during the pre-release window (1-2 months before release) we work of a `prerelease/xxx` branch. We do not push directly on `main` or `prerelease/xxx`, we create PRs on those branches so that CI always run and these important branches are kept in a good state.

Reply:
Thank you! I did merge `main` back into my branch but still seeing some issues with the ID numbers; wondering if those need to be "fixed" manually?

Also, just to confirm I understand this part:
> we create PRs on those branches so that CI always run and these important branches are kept in a good state.
Is this describing how we don't work directly on main, in other words work on a feature, etc. branch? (Sorry I'm not sure how to word this haha. "Normal" git practice...?)

Reply:
Thank you! I did merge main back into my branch but still seeing some issues with the ID numbers; wondering if those need to be "fixed" manually?

Also, just to confirm I understand this part:

> we create PRs on those branches so that CI always run and these important branches are kept in a good state.

Is this describing how we don't work directly on main, in other words work on a feature, etc. branch? (Sorry I'm not sure how to word this haha. "Normal" git practice...?)

Reply:
Yes, this is correct: we work on feature branches that are either based on main or on prerelease branches.

Reply:
Gotcha, thanks for confirming!